instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v1_4/JedisConnectionInstrumentation.java

```diff
@@ -54,12 +68,15 @@ public static class SendCommandNoArgsAdvice {
     @Advice.OnMethodEnter(suppress = Throwable.class)
     public static void onEnter(
         @Advice.This Connection connection,
-        @Advice.Argument(0) Protocol.Command command,
+        @Advice.Argument(0) Object command,
```

Comment:
you could use `@Advice.Argument(value = 0, typing = Assigner.Typing.DYNAMIC) Protocol.Command command`